src/fmu_settings_api/__main__.py

```diff
@@ -8,28 +8,59 @@
 import uvicorn
 from fastapi import FastAPI
 from fastapi.routing import APIRoute
+from fmu.settings._fmu_dir import UserFMUDirectory
+from fmu.settings._init import init_user_fmu_directory
+from fmu.settings._resources.log_manager import LogManager
+from fmu.settings.models.log import Log
+from pydantic import BaseModel
 from starlette.middleware.cors import CORSMiddleware
 
 from .config import HttpHeader, settings
+from .logging import get_logger, setup_logging
+from .middleware.logging import LoggingMiddleware
 from .models import Ok
 from .session import ProjectSession, session_manager
 from .v1.main import api_v1_router
 
 
+class LogEntry(BaseModel):
```

Comment:
```suggestion
class LogEntry(BaseModel, extra="allow"):
```

Another option, since the config is very light this might be a bit nicer